src/e2e-test/features/pipeline.runtimeargs.preview.feature

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Integration_Tests
+Feature: Pipeline - Runtime Args - It must be possible to preview a pipeline with macros
+
+  # Rule: It must be possible to preview a pipeline with macros
+
+    @PIPELINE_RUNTIME_ARGS_TEST
```

Comment:
seems like every line starting here are having 2 extra spaces

Reply:
Leftover from trying to use `Rule`. I'll adjust the files